contents/docs/runbook/upgrade-notes.md

```diff
@@ -4,6 +4,15 @@ sidebar: Docs
 showTitle: true
 ---
 
+### 27.0.0
+
+This version upgrades the bundled `grafana/loki` Helm chart from `2.10.2` to `3.0.3`. The new 3.x Helm Chart 
+is based on the old `grafana/loki-simple-scalable` and as such the configuration options are not compatible.
+As part of this upgrade you'll need to:
+
+1. ensure that your values for `loki:` are compatible with the new chart, see their [README](https://github.com/grafana/loki/tree/main/production/helm/loki) for details. Note that we supply default values that will result in the old "single binary" deployment being deployed, mimicking the old functionality.
+2. delete the old Loki statefulset, as is documented [here](https://github.com/grafana/loki/tree/main/production/helm/loki#upgrading-from-grafanaloki).
+
```

Comment:
Note: if you are not setting `loki.enabled: true` as part of your installation, there's nothing you need to do.

Reply:
Have added a note